tests/platform_tests/api/test_sfp.py

```diff
@@ -718,6 +718,11 @@ def test_reset(self, request, duthosts, enum_rand_one_per_hwsku_hostname, localh
                 continue
             info_dict = port_index_to_info_dict[sfp_port_idx]
 
+            # when an optic XCVR goes through sfp reset on an Arista platform it goes into lpmode
+            # https://github.com/aristanetworks/sonic/issues/121
+            if 'arista' in duthost.facts['platform'].lower() and self.is_xcvr_support_lpmode(info_dict):
```

Comment:
@arista-nwolfe Is this issue seen only with non-CMIS modules?

Per my understanding, this test will currently perform the below actions
1. Reset the module
2. perform shut followed by no shut on the port for CMIS modules - this step will cause XCVRD to bring the module to high power mode

Reply:
We see this problem with both CMIS and non-CMIS XCVRs but I believe the problem is within XCVRD, I'll make the fix there and then update this test to also flap non-CMIS XCVRs.